### ISSUE TITLE
Fail to generate cDNA coordinate for relevant transcript

### ISSUE THREAD

**Issue by AleksandrSl** [biocommons/hgvs#546]
Variant - https://preview.ncbi.nlm.nih.gov/clinvar/variation/25111/ overlaps gene GALT and two of its transcripts -NM_000155.3, NM_001258332.1 - by 1 base. So they are relevant. But `g_to_c ` conversion 
throws `Position is beyond the bounds of transcript record`, and this is reasonable, since not the whole variant is covered by transcipt. Maybe there are some existing workarounds? Or more constrained implementation of `relevant_transcripts` needed so variant must be covered by transcript?

```
self.assembly_mapper = hgvs.assemblymapper.AssemblyMapper(data_provider, assembly_name='GRCh37.p13')
hgvs_parser = hgvs.parser.Parser()
variant = hgvs_parser.parse_hgvs_variant("NC_000009.11:g.34646583_34646586del")
for transcript in self.assembly_mapper.relevant_transcripts(variant):
    logging.warning(self.assembly_mapper.g_to_c(variant, transcript))
```

**Comment by reece** [comment]
@AleksandrSl: I'm unable to reproduce this 

```
(3.6) snafu$ hgvs-shell 
hgvs-shell -- interactive hgvs
hgvs version: 1.2.5.dev13+h6ad6cf213f99
data provider url: postgresql://anonymous:anonymous@uta.biocommons.org/uta/uta_20171026
schema_version: 1.1
data_version: uta_20171026
sequences source: remote (bioutils.seqfetcher)

In [1]: v37 = hp.parse_hgvs_variant("NC_000009.11:g.34646583_34646586del")

In [2]: am37.relevant_transcripts(v37)
Out[2]: []
```

From your question, I think you must be getting at least `NM_000155.3` and `NM_001258332.1` from `relevant_transcripts()`. Would you please retry with `hgvs-shell`? (`hgvs-shell` is installed in your virtual environment when you install `hgvs`.)

uta_20180821 gives the same result.

**Comment by AleksandrSl** [comment]
@reece  Thank you for such a quick reply! 

`pip install hgvs` gives me `1.2.4` version, the latest stable I suppose. 

```
############################################################################
hgvs-shell -- interactive hgvs
hgvs version: 1.2.4
data provider url: postgresql://anonymous:anonymous@uta.biocommons.org/uta/uta_20161216
schema_version: 1.1
data_version: uta_20161216
sequences source: remote (bioutils.seqfetcher)
...
Some text skipped
...

In [1]: v37 = hp.parse_hgvs_variant("NC_000009.11:g.34646583_34646586del")
In [2]: am37.relevant_transcripts(v37)
Out[2]: ['NM_000155.3', 'NM_001258332.1']
```

Same results with `uta_20171026` and `uta_20180821` (isn't included here)
```
############################################################################
hgvs-shell -- interactive hgvs
hgvs version: 1.2.4
data provider url: postgresql://anonymous:anonymous@uta.biocommons.org/uta/uta_20171026
schema_version: 1.1
data_version: uta_20171026
sequences source: remote (bioutils.seqfetcher)
...
Some text skipped
...

In [1]: v37 = hp.parse_hgvs_variant("NC_000009.11:g.34646583_34646586del")

In [2]: am37.relevant_transcripts(v37)
Out[2]: ['NM_000155.3', 'NM_001258332.1']
```

Maybe you have already "fixed" this in your `1.2.5.dev13+h6ad6cf213f99` build.

**Comment by reece** [comment]
Indeed, this was fixed in master. I'll backport it to the 1.2 branch.
Thanks for the report!
